src/include/block_chunkcache.h

```diff
@@ -64,9 +64,10 @@ struct __wt_chunkcache {
     /* Cache-wide. */
 #define WT_CHUNKCACHE_FILE 1
 #define WT_CHUNKCACHE_IN_VOLATILE_MEMORY 2
-    uint8_t type;        /* Location of the chunk cache (volatile memory or file) */
-    uint64_t bytes_used; /* Amount of data currently in cache */
-    uint64_t capacity;   /* Maximum allowed capacity */
+    uint8_t type;               /* Location of the chunk cache (volatile memory or file) */
+    uint64_t bytes_used;        /* Amount of data currently in cache */
+    uint64_t pinned_bytes_used; /* Amount of pinned data currently in cache */
```

Comment:
This doesn't appear to be used - I think it's correct to just have the stat rather than tracking this separately. (The non-pinned version is tracked for eviction purposes.)

Reply:
Sure, updated in [111368a](https://github.com/wiredtiger/wiredtiger/pull/9480/commits/111368ace54fe3cff2b66e2e0b15b1d4d08fcd34)